forge-game/src/main/java/forge/game/trigger/TriggerBecomesTarget.java

```diff
@@ -85,6 +85,7 @@ public final boolean performTest(final Map<AbilityKey, Object> runParams) {
     public final void setTriggeringObjects(final SpellAbility sa, Map<AbilityKey, Object> runParams) {
         sa.setTriggeringObject(AbilityKey.Source, ((SpellAbility) runParams.get(AbilityKey.SourceSA)).getHostCard());
         sa.setTriggeringObjectsFrom(runParams, AbilityKey.SourceSA, AbilityKey.Target);
+        sa.setTriggeringObject(AbilityKey.StackInstance, sa.getHostCard().getGame().getStack().getInstanceFromSpellAbility((SpellAbility) runParams.get(AbilityKey.SourceSA)));
```

Comment:
i hope that getInstanceFromSpellAbility doesn't make us more problems, but i seems to work for now